src/test/java/org/prebid/server/bidder/consumable/ConsumableBidderTest.java

```diff
@@ -1,411 +1,459 @@
+        final BidderCall<BidRequest> httpCall = givenHttpCall(null,
+                mapper.writeValueAsString(null));
 
         // when
-        final Result<List<HttpRequest<ConsumableBidRequest>>> result = target.makeHttpRequests(bidRequest);
+        final CompositeBidderResponse result = target.makeBidderResponse(httpCall, null);
 
         // then
         assertThat(result.getErrors()).isEmpty();
-        assertThat(result.getValue()).hasSize(1)
-                .extracting(httpRequest -> mapper.readValue(httpRequest.getBody(), ConsumableBidRequest.class))
-                .extracting(ConsumableBidRequest::getIncludePricingData,
-                        ConsumableBidRequest::getEnableBotFiltering, ConsumableBidRequest::getParallel)
-                .containsOnly(tuple(true, true, true));
-        assertThat(result.getValue())
-                .extracting(httpRequest -> mapper.readValue(httpRequest.getBody(), ConsumableBidRequest.class))
-                .extracting(ConsumableBidRequest::getTime)
-                .isNotNull();
+        assertThat(result.getBids()).isEmpty();
     }
 
     @Test
-    public void makeHttpRequestsShouldSetRequestFieldsFromFirstImpExt() {
+    public void makeBidderResponseShouldReturnEmptyListIfBidResponseSeatBidIsNull() throws JsonProcessingException {
         // given
-        final BidRequest bidRequest = givenBidRequestWithTwoImpsAndTwoFormats();
+        final BidderCall<BidRequest> httpCall = givenHttpCall(null,
+                mapper.writeValueAsString(BidResponse.builder().build()));
 
         // when
-        final Result<List<HttpRequest<ConsumableBidRequest>>> result = target.makeHttpRequests(bidRequest);
+        final CompositeBidderResponse result = target.makeBidderResponse(httpCall, null);
 
         // then
         assertThat(result.getErrors()).isEmpty();
-        assertThat(result.getValue()).hasSize(1)
-                .extracting(httpRequest -> mapper.readValue(httpRequest.getBody(), ConsumableBidRequest.class))
-                .extracting(ConsumableBidRequest::getNetworkId, ConsumableBidRequest::getSiteId,
-                        ConsumableBidRequest::getUnitId, ConsumableBidRequest::getUnitName)
-                .containsOnly(tuple(111, 222, 333, "unit_name"));
+        assertThat(result.getBids()).isEmpty();
     }
 
     @Test
-    public void makeHttpRequestsShouldReturnRequestWithExpectedPlacements() {
+    public void makeBidderResponseShouldReturnBannerBidIfBannerIsPresent() throws JsonProcessingException {
         // given
-        final BidRequest bidRequest = givenBidRequestWithTwoImpsAndTwoFormats();
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(singletonList(Imp.builder().id("123").banner(Banner.builder().build()).build()))
+                .build();
+        final BidderCall<BidRequest> httpCall = givenHttpCall(
+                bidRequest,
+                mapper.writeValueAsString(
```

Comment:
`mapper.writeValueAsString()` could be hidden inside the `givenBidResponse`

Reply:
Done